app/src/main/java/org/oppia/android/app/activity/route/ActivityRouter.kt

```diff
@@ -16,22 +17,20 @@ class ActivityRouter @Inject constructor(
   private val consoleLogger: ConsoleLogger
 ) {
   /**  Opens the activity corresponding to the specified [destinationScreen]. */
```

Comment:
Nit: extra space before ``Opens``.

Also, this documentation no longer matches what it's doing per the API change.

Reply:
Done